upload-source-maps/action.yml

```
@@ -32,7 +44,7 @@ runs:
 
         if [[ $(yarn davinci-engine help) == *"sentry-upload-source-maps"* ]]
         then
-          yarn davinci-engine sentry-upload-source-maps
+          yarn davinci-engine sentry-upload-source-maps --url-prefix=$URL_PREFIX --app-build-directory=$APP_BUILD_DIRECTORY --release-name=$RELEASE_NAME
```

Because of how quoting works on shell scripts, we need to surround the variable
expansions on quotes, in the case any of them contain spaces:

```suggestion
          yarn davinci-engine sentry-upload-source-maps --url-prefix="$URL_PREFIX" --app-build-directory="$APP_BUILD_DIRECTORY" --release-name="$RELEASE_NAME"
```